### ISSUE TITLE
Multi-class models

### ISSUE THREAD

**Issue by bw4sz** [weecology/DeepForest-pytorch#14]
DeepForest was primarily designed as 'tree' detector, with one class. I think it is useful to atleast allow users to train multi-class models.

Things to investigate 

1. [x] allow multiple labels in the dataset.py
2. [x] have a lookup dict for multiple labels when visualizing
3. [x] Add multiple colors for each class
4. [x] Make sure that evaluation statistics can report multiple classes
5. [x] Make a tutorial is reproducible data in colab
6. [x] Make sure that a reloaded model gets the same classes.

Its possible torch vision has already implemented all of this and just needs to be checked out. I do not yet know.

**Comment by bw4sz** [comment]
Decided against a separate colab tutorial.
